msteams-platform/bots/how-to/authentication/auth-aad-sso-bots.md

```diff
@@ -190,8 +191,8 @@ Start with the [teams auth sample](https://github.com/microsoft/BotBuilder-Sampl
 
 #### Additional code samples
 
-* [C# sample using the Bot Framework SDK](https://microsoft-my.sharepoint-df.com/:u:/p/vul/ETZQfeTViDlCv-frjgTIincB7dvk2HOnma1TLvcoeGGIxg?e=uPq62c).
+* [C# sample using the Bot Framework SDK](/microsoft/BotBuilder-Samples/experimental/teams-sso/csharp_dotnetcore).
```

Comment:
Hi @v-rodriguezd ,

You must include the entire file path because the code lives in the Github repo, not in the Microsoft docset. 
Here is the correct link: 
https://github.com/microsoft/BotBuilder-Samples/tree/main/experimental/teams-sso/csharp_dotnetcore
Please update.
@surbhigupta12